components/builder-originsrv/src/migrations/2018-08-24-184520_initial_originsrv_migration/up.sql

```diff
@@ -0,0 +1,1149 @@
+CREATE SEQUENCE IF NOT EXISTS origin_secrets_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_package_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_channel_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_integration_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_invitations_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_private_encryption_key_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_project_integration_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_project_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_public_key_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_secret_key_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_id_seq;
+CREATE SEQUENCE IF NOT EXISTS origin_public_encryption_key_id_seq;
+
+CREATE TABLE origins (
+    id bigint DEFAULT next_id_v1('origin_id_seq') PRIMARY KEY NOT NULL,
+    name text UNIQUE,
+    owner_id bigint,
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now(),
+    default_package_visibility text DEFAULT 'public'::text NOT NULL
+);
+
+CREATE TABLE audit (
```

Comment:
Do we still use the audit table?  I recall that we deprecated it in lieu of audit_packges/audit_package_groups.  If so, we can remove it (I don't believe it made it into prod, but we should check)

Reply:
I was hoping we'd be able to remove it but I haven't had time to double back to see if it's used